titus-server-master/src/main/java/com/netflix/titus/master/jobmanager/service/DefaultV3JobOperations.java

```diff
@@ -415,6 +417,11 @@ public Completable recordTaskPlacement(String taskId, Function<Task, Task> chang
     @Override
     public Observable<Void> updateJobCapacityAttributes(String jobId, CapacityAttributes capacityAttributes, CallMetadata callMetadata) {
         logger.info("UpdateJobCapacityAttributes for job {} - {}", jobId, capacityAttributes);
+        String callerId = callMetadata.getCallers().isEmpty()
+                ? "unknown"
+                : callMetadata.getCallers().get(0).getId();
+        updateJobAttributes(jobId, ImmutableMap.of(JobAttributes.JOB_ATTRIBUTES_CREATED_BY, callerId,
```

Comment:
This is not doing anything, as it is `Mono` that must be subscribed. But this method should not be called from here. Instead you should update `BasicServiceJobActions.updateJobCapacityAction`. Call metadata is passed there.